www/src/data/intl/en.yaml

```diff
@@ -0,0 +1,17 @@
+docs.tableOfContents: Table of Contents
```

Comment:
Trying to decide if we should support [nested keys](https://github.com/wiziple/gatsby-plugin-intl/blob/master/examples/gatsby-starter-default-intl/src/intl/en.json) like [gatsby-plugin-intl](https://www.gatsbyjs.org/packages/gatsby-plugin-intl/?=gatsby-plugin-intl). The implementation shouldn't be [too difficult](https://github.com/wiziple/gatsby-plugin-intl/blob/master/src/gatsby-node.js#L3). Regardless, it would be good to have some sort of namespacing to delineate different parts of the website (docs vs search vs feedback widget).